server/account/account.go

```diff
@@ -94,71 +94,79 @@ const (
 	// NoRule indicates that no rules have been broken. This may be an invalid
 	// value in some contexts.
 	NoRule Rule = iota
+	// PreimageReveal means an account failed to respond with a valid preimage
+	// for their order during epoch processing.
+	PreimageReveal
 	// FailureToAct means that an account has not followed through on one of their
 	// swap negotiation steps.
-	PreimageReveal
-	// MaxRule in not an actual rule. It is a placeholder that is used to
-	// determine the total number of rules. It must always be the last
-	// definition in this list.
 	FailureToAct
 	// CancellationRatio means the account's cancellation ratio has dropped below
 	// the acceptable level.
 	CancellationRatio
 	// LowFees means an account made a transaction that didn't pay fees at the
 	// requisite level.
 	LowFees
-	// PreimageReveal means an account failed to respond with a valid preimage
-	// for their order during epoch processing.
+	// MaxRule in not an actual rule. It is a placeholder that is used to
+	// determine the total number of rules. It must always be the last
+	// definition in this list.
 	MaxRule
 )
 
-// ruleNames is a map of rules to names.
-var ruleNames = map[Rule]string{
-	NoRule:            "NoRule",
-	PreimageReveal:    "PreimageReveal",
-	FailureToAct:      "FailureToAct",
-	CancellationRatio: "CancellationRatio",
-	LowFees:           "LowFees",
+// details holds rule specific details.
+type details struct {
+	name, details string
```

Comment:
rule `description` sounds better to me than rule `details`, what you think?

Reply:
Ok! Actually paused for quite a while trying to think of a name other than details, lol.